### PR TITLE
5-database-schema-and-seed

### PR DESCRIPTION
**Pull request by mdwiltfong**
This PR will enhance the apps ability to create and seed databases in the local enviornment.

At the time of making this PR, the tests were not passing

### CODE REVIEW

**Comment by mdwiltfong** [comment]
Hey @hazeltonbw ! Let me know what you think of this PR. Hopefully you can drop by tonight for the JS Support hour. I wasn't able to run this PR through your tests, because despite running `jest <testname>` it looked like they were all failing. I did some manual testing and it looks gucci. Nonetheless I wouldn't merge it until the tests pass. 

Also, I noticed that the GET request wasn't returning a cookie. I'll need to look more into that

File: .vscode/settings.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   "files.simpleDialog.enable": true,
   "workbench.fontAliasing": "antialiased",
   "workbench.iconTheme": "vs-seti",
-  "workbench.sideBar.location": "right",
+  "workbench.sideBar.location": "left",
```
Comment (hazeltonbw) [inline .vscode/settings.json:26]: I tried to work with my sidebar on the right but I just don't like it. I agree with this change

File: config.js [hunk 2]
```diff
@@ -1,11 +1,17 @@
+/*
+@hazeltonbw When running `npm run create-db` there are no enviornment variables.
+By using the npm package `dotenv` we can use the .env file at root. You may not want to do it this way,
+but either way the npm script doesn't seem to be picking up the .env in root.
+*/
+require("dotenv").config();
```
Comment (hazeltonbw) [inline config.js:6]: This seems better than having to specify how to use the dotenv file in your package.json scripts

File: db/setupDatabase.js [hunk 3]
```diff
@@ -52,6 +52,7 @@ const { DB } = require("../config");
 
   try {
     // Make a temporary client so we can create tables in the database
+    console.log(DB);
```
Comment (hazeltonbw) [inline db/setupDatabase.js]: Forgot to turn off logging?

File: .vscode/settings.json (hunk 1 above)
Comment (mdwiltfong) [reply .vscode/settings.json:26]: I agree! Although, I'm confused how this is getting picked up by git 😂

File: db/setupDatabase.js (hunk 3 above)
Comment (mdwiltfong) [reply db/setupDatabase.js]: Agh, yes. Thank you for catching that. 

This is a good example for static testing. Maybe something like ESLint could pick up mistakes like this.

Comment (mdwiltfong) [reply db/setupDatabase.js]: I'm seeing this line of code in the PR, although only in a previous commit. I think the line of code was deleted with the latest commit.

**Review by hazeltonbw — APPROVED** [review]

**Pull request merged**